app/localize.js

```diff
@@ -0,0 +1,27 @@
+var languages = {
+    "en_US": {
+        "name": "English", "messages": require.main.require("./locales/en_US.json")
```

Comment:
This is not really important, but language names can also be automatically found from lang code by using the `langmap` module which is already a dependency (see an example in https://github.com/brrd/Abricotine/blob/master/app/abr-menu.js#L78)